hurumap/settings.py

```diff
@@ -103,30 +99,65 @@
 HURUMAP['map_zoom'] = 6
 
 
+HURUMAP['geodata'] = 'hurumap.geo.GeoData'
+HURUMAP['mapit'] = {}
+HURUMAP['geometry_data'] = {
+    '2009': {
+        'country': 'geo/country.topojson',
+        'county': 'geo/county.topojson'
+
+    }
+}
+
+HURUMAP['USE_MAPIT'] = os.environ.get('USE_MAPIT', False)
```

Comment:
Whatever comes from `os.environ.get` is a string. You need to do cast to boolean or `if HURUMAP['USE_MAPIT'] == "True"`